tests/e2e/test_chaperone.js

```diff
@@ -297,16 +258,17 @@ describe("Server", () => {
         }
 
         try {
-          return client.callZomeFunction('test-hha', "hha", "get_happ", client.hha_hash);
+          // Note: the cell_id is `elemental-chat.dna.gz` because holochain-run-dna is setting a default nick
+          // Ideally we would have a nick like elemental-chat or chat
+          return client.callZomeFunction(`elemental-chat.dna.gz`, "chat", "agent_stats", {});
```

Comment:
Why is the dna-nick a file name?

Reply:
I have written a note above. That is the nick `holochain-run-dna` uses. We would need to update holochain-run-dna to take in a nick. Until then we will have that as a nick